src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -593,7 +595,11 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 						m_user[it->first] = data;
 					}
 				}
-				if( boost::starts_with( it->first.string(), g_lightFilterPrefix.string() ) )
+				else if( it->first == g_sssSetNameName )
```

Comment:
Let's move this out of the loop and do a single lookup with `attribute<IECore::InternedStringVectorData>( g_sssSetNameName, attributes )` above, as we have for all the others. This has the advantage of being less verbose and reporting any invalid data types to the user, and I think it also reduces the number of comparisons...